gaphor/codegen/codegen.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""The Gaphor code generator CLI.
+
+Provides the CLI for the code generator which transforms a Gaphor models (with
+.gaphor file extension) in to a data model in Python.
+"""
+
+import argparse
+from distutils.util import byte_compile
+from pathlib import Path
+
+from gaphor.codegen import autocoder
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "modelfile",
+        type=Path,
+        help="gaphor model filename, default location is the "
+        "models package if full filename not given.",
+    )
+    parser.add_argument(
+        "outfile",
+        type=Path,
+        help="python data model filename, default location is "
+        "the model name package if full filename not "
+        "given.",
+    )
+    parser.add_argument(
+        "overrides",
+        type=Path,
+        help="override filename, default location is the "
+        "models package if full filename not given.",
+    )
+    args = parser.parse_args()
+    modelfile: Path = args.modelfile
+    outfile: Path = args.outfile
+    if str(modelfile) == modelfile.name:
```

Comment:
I think we do not need the `if`-blocks. Let's just be specific in the arguments instead. If we can not resolve a file, just exit.

Reply:
Ok, I'm good with that! :smile_cat: